Client/Application/main.swift

```diff
@@ -10,11 +10,11 @@ if AppConstants.IsRunningTest || AppConstants.IsRunningFastlaneSnapshot {
     appDelegate = TestAppDelegate.self
 } else {
     switch AppConstants.BuildChannel {
-    case .Aurora:
```

Comment:
Are we getting rid of Aurora completely? Or will Aurora still exist, it wll just not use the AuroraAppDelegate anymore?